CmrUI/Helpers/GradientHelper.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  GradientHelper.swift
+//  CmrUI
+//
+//  Created by Vasyl.Savka on 7/19/18.
+//  Copyright © 2018 Vasyl.Savka. All rights reserved.
+//
+
+import UIKit
+import ObjectiveC
+
+private var GradientLayerKey: UInt8 = 0
+
+class GradientHelper: NSObject {
```

Comment:
@vmsavka If we're receiving a UIView in all calls to this class, then it is a UIView extension. Also, if the gradient is the same, there is no need to associate the object, just use a private var privateGradientLayer: CALayer

Reply:
fixed